internal/provider/resource_cloudflare_access_organization_test.go

```diff
@@ -0,0 +1,77 @@
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccCloudflareAccessOrganization(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("cloudflare_access_organization.%s", rnd)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPreCheck(t)
+			testAccessAccPreCheck(t)
+		},
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{
+			{
+				Config:           testAccCloudflareAccessOrganizationConfigBasic(rnd, accountID),
+				ResourceName:     name,
+				ImportState:      true,
+				ImportStateId:    accountID,
+				ImportStateCheck: accessOrgImportStateCheck,
+			},
+		},
+	})
+}
+
+func accessOrgImportStateCheck(instanceStates []*terraform.InstanceState) error {
+	state := instanceStates[0]
+	attrs := state.Attributes
+
+	stateChecks := []struct {
+		field         string
+		stateValue    string
+		expectedValue string
+	}{
+		{field: "ID", stateValue: state.ID, expectedValue: accountID},
+		{field: "account_id", stateValue: attrs["account_id"], expectedValue: accountID},
+		{field: "name", stateValue: attrs["name"], expectedValue: "my test org"},
+		{field: "auth_domain", stateValue: attrs["auth_domain"], expectedValue: "authdomain.cloudflareaccess.com"},
```

Comment:
I added dummy data here but we'll need to update these test cases for the account that's used in CI.